superset/config.py

```diff
@@ -1328,6 +1328,8 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 # Slack API token for the superset reports, either string or callable
 SLACK_API_TOKEN: Callable[[], str] | str | None = None
 SLACK_PROXY = None
+# Whether Superset should use Slack avatars for users
+SLACK_ENABLE_AVATARS = True
```

Comment:
At one point we discussed and _might have completed_ moving all boolean configs to Feature Flags. If that's going to be a policy (which I don't think has _official_ consensus) then this should be too.  CC @eschutho and @michael-s-molina in case they remember differently. I'm just a fan of consistency, in any case ;)

Reply:
We need to have a discussion about feature flags in a specific meeting. I was researching a little bit about feature flags best practices and [this](https://docs.getunleash.io/topics/feature-flags/short-lived-feature-flags) caught my attention. Specifically, the following:

> The most common use case for feature flags is to protect new functionality. That means that when the roll-out of new functionality is complete, the feature flag should be removed from the code and archived.

> Feature flags should not be used for static application configuration.

If we followed a similar model, we would never have the categorization we currently have in [FEATURE_FLAGS.md](https://github.com/apache/superset/blob/master/RESOURCES/FEATURE_FLAGS.md). We would just assume that all feature flags should be removed once the feature is complete and what we have as `STABLE` would actually be application configurations, and ideally [changeable at runtime](https://docs.getunleash.io/topics/feature-flags/runtime-control). What I like about this model is the clarity/simplicity.

Reply:
Yes there's a fine line between configuration setting and feature flag. Happy to move to FF if we think it belong there